packages/cli/src/extensions/plugins-extensions.ts

```diff
@@ -63,7 +63,7 @@ module.exports = (toolbox: GluegunToolbox) => {
         : `${toolbox.normalizeBaseUrl(
             toolbox.inputParameters.shopwareEndpoint
           )}/${config}`;
-    const pluginsConfigResponse = await axios.get(url);
+    const pluginsConfigResponse = await axios.get(url.replace('//', '/'));
```

Comment:
In that case url:
`http://something.com//some/link`
would become:
`http:/something.com/some/link`

which would not be correct

Reply:
*welp* for sure you are correct, ill update the PR. it was just a quick edit through the Github UI ;)